internal/plugin/config_plugin.go

```diff
@@ -75,6 +77,18 @@ func (c *Config) Process(ctx context.Context, msg *bus.Message) {
 			c.resource = resource
 			c.resourceMutex.Unlock()
 		}
+
+		once.Do(func() {
+			// This will be replaced when we implement config upload
+			instanceList := c.resource.GetInstances()
+			if len(instanceList) > 1 {
```

Comment:
I assume this is because we treat the Agent itself as an instance, so the second element is an NGINX instance?

Reply:
Yes, need to check there is more than one as the Agent is an instance in this case